commands/playlist.ts

```diff
@@ -50,6 +50,8 @@ export default {
       });
 
       bot.queues.set(message.guild!.id, newQueue);
+      newQueue.songs.push(...playlist.videos);
+      
       newQueue.enqueue(playlist.videos[0]);
```

Comment:
This adds the first song in the playlist into the queue twice.

Reply:
Hm, could the enqueue part be removed since i’m pushing the songs, or no?

Reply:
Actually, upon testing that is false. It does not add the first song twice.